packages/lit-dev-content/site/blog/2023-10-10-lit-3.0.md

```diff
@@ -0,0 +1,262 @@
+
+The new decorator spec has reached Stage 3 in TC39, meaning that browsers and compilers are now implementing them. This is a huge step for Lit! The arrival of standard decorators allows us to begin the process of moving to a decorator implementation that won't require a compiler to use.
+
+It's very important to us to make the upgrade path from experimental decorators as smooth as possible. To accomplish this we made our existing decorators support the standard spec and made them work with the new `accessor` keyword in experimental decorator mode.
+
+This way you can use Lit decorators with auto-accessors (using the `accessor` keyword) so that the same call site works with both settings:
+
+```ts
+class MyElement extends LitElement {
+  @property()
+  accessor myProperty = 'hello';
+}
+```
+
+Once all decorator call sites in your project use the `accessor` keyword, you can build with `experimentalDecorators` set to `true` or `false` without a change in behavior.
+
+But in order to make these hybrid decorators have consistent behavior in both modes we had to make a few minor breaking changes to our experimental decorators:
+
+- We now call `requestUpdate()` automatically for `@property` and `@state` decorated accessors where previously that was the setter's responsibility.
+- The value of an accessor is read on first render and used as the initial value for `changedProperties` and attribute reflection.
+- `@property` and `@state` must be used on the setter for hand-written accessors.
+
+Standard decorator mode requires [TypeScript 5.2](https://devblogs.microsoft.com/typescript/announcing-typescript-5-2/#decorator-metadata) or [Babel 7.23](https://babeljs.io/blog/2023/09/25/7.23.0) with the [`@babel/plugin-proposal-decorators`](https://babeljs.io/docs/babel-plugin-proposal-decorators) plugin using decorator version `"2023-05"`.
+
+### Upgrading
+
+The upgrade from Lit 2.0 should be seamless for the vast majority of users. You can usually upgrade your npm dependency version with:
+
+```sh
+> npm i lit@latest
+```
+
+You can find more details and how to handle the changes in the [Lit 3 upgrade guide](/docs/v3/releases/upgrade/).
+
+Detailed change logs can be found [on GitHub](https://github.com/lit/lit/releases?q=%22-pre.1%22&expanded=true).
+
+## Even Faster Rendering with the new Lit Template Compiler { #compiler }
+
+The Lit team is excited to announce a new Labs package [`@lit-labs/compiler`](https://www.npmjs.com/package/@lit-labs/compiler), which provides a [TypeScript Transformer](https://github.com/itsdouges/typescript-transformer-handbook#the-basics) that can be run over your JavaScript or TypeScript files to do build-time preparation  of Lit templates that Lit would normally do at runtime.
+
+![Lit Compiler benchmarks](/images/blog/3.0-launch/compiler-benchmarks.png)
+
+In a best case scenario with a template heavy page, we measured a 46% faster first render, and a 21% faster update!
```

Comment:
```suggestion
In a benchmark that featured large and complicated templates, we measured a 46% faster first render, and a 21% faster update!
```

Reply:
The previous phrasing hedged things a bit, can we keep expectations in check with a range? Maybe "While not all templates will see rendering performance improvements from compilation, on our "template heavy" benchmark we measured..."